src/ocl/batchnormocl.cpp

```diff
@@ -976,8 +976,8 @@ void BatchNormBackward(Handle& handle,
                 }
                 else
                 {
-                    xlocalsize = 256;
-                    xgridsize  = 256 * c;
+                    xlocalsize = 512;
+                    xgridsize  = 512 * c;
```

Comment:
I have concern of this change.
The root cause of the issue is failing to correctly interpret data in float4 type, not the size of the work group.
Enlarging the group size could cover some specific test cases, but will not solve the root issue.

Reply:
@ce1adon Even if so -- is this a good change? Or does it give nothing provided that the root reason is fixed properly?

Reply:
It gives nothing to fix the root reason.
The only impact is in performance. Will have to wait for @muralinr 's conclusion for this impact.

Reply:
@ce1adon Thanks.

Reply:
@atamazov Please note that enlarge the group size could lead to summation overflow in fp16, regarding to the issue you are working on.